src/components/Table/Budget/BudgetLineItemTable/BudgetLineItemTableContainer.tsx

```diff
@@ -68,7 +68,7 @@ function BudgetLineItemTableContainer({
 
 	const toggleDialogs = (index: number, val: boolean) => {
```

Comment:
what is `val`?

Reply:
value is the value we want to set

Reply:
It could be renamed to a meaningful name